consensus/hotstuff/votecollector/statemachine_test.go

```diff
@@ -90,9 +90,9 @@ func (s *StateMachineTestSuite) TestStatus_StateTransitions() {
 	require.Equal(s.T(), hotstuff.VoteCollectorStatusVerifying, s.collector.Status())
 
 	// after submitting double proposal we should transfer into invalid state
-	err = s.collector.ProcessBlock(helper.MakeProposal(
-		helper.WithBlock(
-			helper.MakeBlock(helper.WithBlockView(s.view)))))
+	err = s.collector.ProcessBlock(helper.MakeSignedProposal(helper.WithProposal(
```

Comment:
can we use the function `makeSignedProposalWithView` here too? Just skip this suggestion if we can't. No detailed explanation needed.